src/core/storage/compression/Decompressor.h

```diff
@@ -135,9 +135,9 @@ class DecompressorImpl {
 	/// \brief process_byte_number reads 8 successive bits from compressed file
 	/// (does not have to be in the same byte)
 	///
-	/// \return the latter 8 successive bits in unsigned char form
-	unsigned char process_byte_number() {
-		unsigned char val, temp_byte;
+	/// \return the latter 8 successive bits in uint8_t form
+	uint8_t process_byte_number() {
+		uint8_t val, temp_byte;
```

Comment:
Please separate [declarations on multiple lines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es10-declare-one-name-only-per-declaration).

Reply:
Aaah, this separate lines :). Ok, fixed!